### ISSUE TITLE
Release 2.0.9 with msp430-elf-gcc 9.2.0.50 fails to build the `Blink` example against the MSP430FR5994.

### ISSUE THREAD

**Issue by rei-vilo** [energia/msp430-lg-core#131]
Release 2.0.9 with msp430-elf-gcc 9.2.0.50 fails to build the `Blink` example against the MSP430FR5994.

```
/Users/USER/Library/Energia15/packages/energia/tools/msp430-elf-gcc/9.2.0.50/bin/msp430-elf-gcc -c -g -Os -w -ffunction-sections -fdata-sections -MMD -mmcu=msp430fr5994 -DF_CPU=16000000L -DARDUINO=10811 -DENERGIA=10811 -DENERGIA_MSP_EXP430FR5994LP -DENERGIA_ARCH_MSP430ELF -I/Users/USER/Library/Energia15/packages/energia/tools/msp430-elf-gcc/9.2.0.50/include -I/Users/USER/Library/Energia15/packages/energia/tools/msp430-elf-gcc/9.2.0.50/msp430-gcc-support-files/include -mlarge -mcode-region=either -mhwmult=auto -I/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/cores/msp430 -I/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP /Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/cores/msp430/wiring_pulse.c -o /var/folders/wn/n7qqb8ss0k3bvpqwfcdwp_7r0000gn/T/arduino_build_223338/core/wiring_pulse.c.o
In file included from /Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/cores/msp430/Energia.h:366,
                 from /Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/cores/msp430/wiring_private.h:37,
                 from /Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/cores/msp430/wiring_digital.c:33:
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:246:23: error: 'P2_BASE' undeclared here (not in a function); did you mean 'P1_BASE'?
  246 |   #define P2_X_BASE   P2_BASE
      |                       ^~~~~~~
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:262:16: note: in expansion of macro 'P2_X_BASE'
  262 |    (uint16_t) (P2_X_BASE+OFS_P2IN),
      |                ^~~~~~~~~
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:295:4: error: initializer element is not constant
  295 |    (uint16_t) (P2_X_BASE+OFS_P2OUT),
      |    ^
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:295:4: note: (near initialization for 'port_to_output[2]')
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:328:4: error: initializer element is not constant
  328 |    (uint16_t) (P2_X_BASE+OFS_P2DIR),
      |    ^
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:328:4: note: (near initialization for 'port_to_dir[2]')
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:361:5: error: initializer element is not constant
  361 |     (uint16_t) (P2_X_BASE+OFS_P2REN),
      |     ^
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:361:5: note: (near initialization for 'port_to_ren[2]')
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:394:5: error: initializer element is not constant
  394 |     (uint16_t) (P2_X_BASE+OFS_P2SEL0),
      |     ^
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:394:5: note: (near initialization for 'port_to_sel0[2]')
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:427:5: error: initializer element is not constant
  427 |     (uint16_t) (P2_X_BASE+OFS_P2SEL1),
      |     ^
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:427:5: note: (near initialization for 'port_to_sel1[2]')
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:516:4: error: 'P6' undeclared here (not in a function); did you mean 'A6'?
  516 |    P6,        /* 3  - P6.1 */
      |    ^~
      |    A6
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:516:4: error: initializer element is not constant
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:516:4: note: (near initialization for 'digital_pin_to_port[3]')
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:517:4: error: initializer element is not constant
  517 |    P6,        /* 4  - P6.0 */
      |    ^~
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:517:4: note: (near initialization for 'digital_pin_to_port[4]')
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:518:4: error: initializer element is not constant
  518 |    P6,        /* 5  - P6.2 */
      |    ^~
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:518:4: note: (near initialization for 'digital_pin_to_port[5]')
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/cores/msp430/WInterrupts.c: In function 'attachInterrupt':
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/cores/msp430/WInterrupts.c:132:10: error: 'P3' undeclared (first use in this function); did you mean 'A3'?
  132 |     case P3:
      |          ^~
      |          A3
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/cores/msp430/WInterrupts.c:132:10: note: each undeclared identifier is reported only once for each function it appears in
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/cores/msp430/WInterrupts.c:145:10: error: 'P4' undeclared (first use in this function); did you mean 'A4'?
  145 |     case P4:
      |          ^~
      |          A4
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:520:4: error: 'P5' undeclared here (not in a function); did you mean 'A5'?
  520 |    P5,        /* 7  - P5.2 */
      |    ^~
      |    A5
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:520:4: error: initializer element is not constant
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:520:4: note: (near initialization for 'digital_pin_to_port[7]')
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:521:4: error: initializer element is not constant
  521 |    P6,        /* 8  - P6.3 */
      |    ^~
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:521:4: note: (near initialization for 'digital_pin_to_port[8]')
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/cores/msp430/WInterrupts.c:158:10: error: 'P5' undeclared (first use in this function); did you mean 'A5'?
  158 |     case P5:
      |          ^~
      |          A5
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:522:4: error: 'P7' undeclared here (not in a function); did you mean 'A7'?
  522 |    P7,        /* 9  - P7.1 */
      |    ^~
      |    A7
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:522:4: error: initializer element is not constant
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:522:4: note: (near initialization for 'digital_pin_to_port[9]')
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:523:4: error: initializer element is not constant
  523 |    P7,        /* 10 - P7.0 */
      |    ^~
/Users/USER/Library/Energia15/packages/energia/tools/msp430-elf-gcc/9.2.0.50/bin/msp430-elf-gcc -c -g -Os -w -ffunction-sections -fdata-sections -MMD -mmcu=msp430fr5994 -DF_CPU=16000000L -DARDUINO=10811 -DENERGIA=10811 -DENERGIA_MSP_EXP430FR5994LP -DENERGIA_ARCH_MSP430ELF -I/Users/USER/Library/Energia15/packages/energia/tools/msp430-elf-gcc/9.2.0.50/include -I/Users/USER/Library/Energia15/packages/energia/tools/msp430-elf-gcc/9.2.0.50/msp430-gcc-support-files/include -mlarge -mcode-region=either -mhwmult=auto -I/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/cores/msp430 -I/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP /Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/cores/msp430/wiring_shift.c -o /var/folders/wn/n7qqb8ss0k3bvpqwfcdwp_7r0000gn/T/arduino_build_223338/core/wiring_shift.c.o
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:523:4: note: (near initialization for 'digital_pin_to_port[10]')
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/cores/msp430/WInterrupts.c:171:10: error: 'P6' undeclared (first use in this function); did you mean 'A6'?
  171 |     case P6:
      |          ^~
      |          A6
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/cores/msp430/WInterrupts.c:184:10: error: 'P7' undeclared (first use in this function); did you mean 'A7'?
  184 |     case P7:
      |          ^~
      |          A7
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/cores/msp430/WInterrupts.c:197:10: error: 'P8' undeclared (first use in this function); did you mean 'A8'?
  197 |     case P8:
      |          ^~
      |          A8
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/cores/msp430/WInterrupts.c: In function 'detachInterrupt':
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/cores/msp430/WInterrupts.c:276:7: error: 'P3' undeclared (first use in this function); did you mean 'A3'?
  276 |  case P3:
      |       ^~
      |       A3
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:524:4: error: 'P8' undeclared here (not in a function); did you mean 'A8'?
  524 |    P8,        /* 11 - P8.1 */
      |    ^~
      |    A8
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:524:4: error: initializer element is not constant
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:524:4: note: (near initialization for 'digital_pin_to_port[11]')
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:525:4: error: initializer element is not constant
  525 |    P8,        /* 12 - P8.2 */
      |    ^~
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:525:4: note: (near initialization for 'digital_pin_to_port[12]')
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:526:4: error: initializer element is not constant
  526 |    P8,        /* 13 - P8.3 */
      |    ^~
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:526:4: note: (near initialization for 'digital_pin_to_port[13]')
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:527:4: error: initializer element is not constant
  527 |    P5,        /* 14 - P5.1 */
      |    ^~
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:527:4: note: (near initialization for 'digital_pin_to_port[14]')
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:528:4: error: initializer element is not constant
  528 |    P5,        /* 15 - P5.0 */
      |    ^~
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:528:4: note: (near initialization for 'digital_pin_to_port[15]')
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:530:4: error: initializer element is not constant
  530 |    P5,        /* 17 - P5.3 */
      |    ^~
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:530:4: note: (near initialization for 'digital_pin_to_port[17]')
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/cores/msp430/WInterrupts.c:283:7: error: 'P4' undeclared (first use in this function); did you mean 'A4'?
  283 |  case P4:
      |       ^~
      |       A4
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:531:4: error: 'P4' undeclared here (not in a function); did you mean 'A4'?
  531 |    P4,        /* 18 - P4.4 */
      |    ^~
      |    A4
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:531:4: error: initializer element is not constant
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:531:4: note: (near initialization for 'digital_pin_to_port[18]')
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:532:4: error: initializer element is not constant
  532 |    P5,        /* 19 - P5.7 */
      |    ^~
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:532:4: note: (near initialization for 'digital_pin_to_port[19]')
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:536:4: error: 'P3' undeclared here (not in a function); did you mean 'A3'?
  536 |    P3,        /* 23 - P3.0 */
      |    ^~
      |    A3
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:536:4: error: initializer element is not constant
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:536:4: note: (near initialization for 'digital_pin_to_port[23]')
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:537:4: error: initializer element is not constant
  537 |    P3,        /* 24 - P3.1 */
      |    ^~
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:537:4: note: (near initialization for 'digital_pin_to_port[24]')
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:538:4: error: initializer element is not constant
  538 |    P3,        /* 25 - P3.2 */
      |    ^~
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:538:4: note: (near initialization for 'digital_pin_to_port[25]')
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:539:4: error: initializer element is not constant
  539 |    P3,        /* 26 - P3.3 */
      |    ^~
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:539:4: note: (near initialization for 'digital_pin_to_port[26]')
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:542:4: error: initializer element is not constant
  542 |    P4,        /* 29 - P4.7 */
      |    ^~
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:542:4: note: (near initialization for 'digital_pin_to_port[29]')
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:543:4: error: initializer element is not constant
  543 |    P8,        /* 30 - P8.0 */
      |    ^~
/Users/USER/Library/Energia15/packages/energia/tools/msp430-elf-gcc/9.2.0.50/bin/msp430-elf-gcc -c -g -Os -w -ffunction-sections -fdata-sections -MMD -mmcu=msp430fr5994 -DF_CPU=16000000L -DARDUINO=10811 -DENERGIA=10811 -DENERGIA_MSP_EXP430FR5994LP -DENERGIA_ARCH_MSP430ELF -I/Users/USER/Library/Energia15/packages/energia/tools/msp430-elf-gcc/9.2.0.50/include -I/Users/USER/Library/Energia15/packages/energia/tools/msp430-elf-gcc/9.2.0.50/msp430-gcc-support-files/include -mlarge -mcode-region=either -mhwmult=auto -I/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/cores/msp430 -I/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP /Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/cores/msp430/avr/dtostrf.c -o /var/folders/wn/n7qqb8ss0k3bvpqwfcdwp_7r0000gn/T/arduino_build_223338/core/avr/dtostrf.c.o
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:543:4: note: (near initialization for 'digital_pin_to_port[30]')
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:544:4: error: initializer element is not constant
  544 |    P4,        /* 31 - P4.1 */
      |    ^~
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:544:4: note: (near initialization for 'digital_pin_to_port[31]')
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:545:4: error: initializer element is not constant
  545 |    P4,        /* 32 - P4.2 */
      |    ^~
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:545:4: note: (near initialization for 'digital_pin_to_port[32]')
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:546:4: error: initializer element is not constant
  546 |    P4,        /* 33 - P4.3 */
      |    ^~
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:546:4: note: (near initialization for 'digital_pin_to_port[33]')
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:549:4: error: initializer element is not constant
  549 |    P7,        /* 36 - P7.3 */
      |    ^~
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:549:4: note: (near initialization for 'digital_pin_to_port[36]')
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:550:4: error: initializer element is not constant
  550 |    P3,        /* 37 - P3.4 */
      |    ^~
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:550:4: note: (near initialization for 'digital_pin_to_port[37]')
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:551:4: error: initializer element is not constant
  551 |    P3,        /* 38 - P3.5 */
      |    ^~
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:551:4: note: (near initialization for 'digital_pin_to_port[38]')
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:552:4: error: initializer element is not constant
  552 |    P3,        /* 39 - P3.6 */
      |    ^~
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:552:4: note: (near initialization for 'digital_pin_to_port[39]')
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:553:4: error: initializer element is not constant
  553 |    P3,        /* 40 - P3.7 */
      |    ^~
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:553:4: note: (near initialization for 'digital_pin_to_port[40]')
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:558:4: error: initializer element is not constant
  558 |    P5,        /* 45 - P5.6 */
      |    ^~
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:558:4: note: (near initialization for 'digital_pin_to_port[45]')
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:559:4: error: initializer element is not constant
  559 |    P5,        /* 46 - P5.5 */
      |    ^~
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:559:4: note: (near initialization for 'digital_pin_to_port[46]')
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:560:4: error: initializer element is not constant
  560 |    P4,        /* 47 - P4.0 */
      |    ^~
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:560:4: note: (near initialization for 'digital_pin_to_port[47]')
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:564:4: error: initializer element is not constant
  564 |    P7,        /* 51 - P7.2 */
      |    ^~
/Users/USER/Library/Energia15/packages/energia/hardware/msp430elf/2.0.9/variants/MSP-EXP430FR5994LP/pins_energia.h:564:4: note: (near initialization for 'digital_pin_to_port[51]')
exit status 1
Error compiling for board MSP-EXP430FR5994LP.
```

Using legacy release 1.0.7 with msp430-gcc 4.6.6 succeeds.

**Comment by rei-vilo** [comment]
@StefanSch Using previous releases 8.3.0.16 or 8.3.1.25 of msp430-elf-gcc doesn't raise the issue.

**Comment by StefanSch** [comment]
Thanks for finding. 
Unfortunately the header files in the latest release from TI got corrupted or some information got removed.
I just made another release with the previous version of header files 1.209
Also the previous package of energia with elf compiler bases on compiler version 8 is working (energia version 2.0.9).
So now the energia version 2.0.10 has compiler version 9.x but uses the previous header package 1.209
